src/riak_kv_qry_compiler.erl

```diff
@@ -22,11 +22,9 @@
 %% -------------------------------------------------------------------
 -module(riak_kv_qry_compiler).
 
--export([
-         compile/3,
-         run_select/2,
-         run_select/3  %% what is this for?
-        ]).
+-export([compile/3]).
+-export([finalise_aggregate/2]).
+-export([run_select/2, run_select/3]).
 
```

Comment:
It makes it very easy to sort alphabetically and group functions with the same name but different arity.